CMSgov/onemac
# OY2 21350: Issues When exporting to the User Management Screen

Story: https://qmacbis.atlassian.net/browse/OY2-21350
Endpoint: See github-actions bot comment

### Details
when the dates were switched from seconds to milliseconds, the seed data and the export to CSV were missed.

### Changes
- removed the * 1000 in the export, since data is now stored in milliseconds
- updated seed data to use milliseconds for user date (basically a changeDate)

### Test Plan
1. Login as a user with user management abilities
2. Navigate to User Management Dashboard
3. Verify the Dates shown
4. Click "Export to CSV"
5. Open the CSV
6. Verify the Dates
